src/pages/learn.js

```diff
@@ -69,19 +69,17 @@ const HeaderContent = styled('div')`
     margin-right: auto;
 `;
 
-const CardContainer = styled('div')`
+const Article = styled('article')`
     padding: ${size.medium};
 `;
 
 const StyledFilterBar = styled(FilterBar)`
-    margin: 0 auto;
+    margin: 0 34px;
```

Comment:
Are these px coming from the design? If so can we just use `large` from theme.js? How does that look?

Reply:
They're not. I was just trying out different px to align the cards, filter, and tab component based on the design. I tried using `large` from theme.js first, but things weren't perfectly aligned. If that's not a hard requirement, I can go ahead and change it to `large`.

Reply:
Yeah let's try to keep to theme.js, I spoke to Dan a bit about which sizes we have for the future so let's use `large` here (or something from `theme`) if possible!